README.md

```diff
@@ -1,2 +1,31 @@
 # comp110-worksheet-B
```

Comment:
Flowchart is good. Could be improved by making "attempts" more explicit -- i.e. showing a variable being initialised, decremented on each guess, and checked.